autopush/src/client.rs

```diff
@@ -518,7 +515,6 @@ where
                     Ok(ServerMessage::Hello {
                         uaid: uaid.as_simple().to_string(),
                         status: 200,
-                        use_webpush: Some(true),
```

Comment:
This needs to be put back like we just did for autoconnect

Reply:
If we are currently 100% on autoconnect, shouldn't we drop the code related to autopush entirely? 

I'm fine reverting this change here, but it's more for minor historical reasons.

Reply:
We're definitely dropping it at [some point](https://mozilla-hub.atlassian.net/browse/SYNC-4036), I'd like to wait a little bit before doing so just to be on the safe side.

I'm also fine w/ 100% reverting everything in the autopush crate on this PR and pretty much not touching anything in there going forward.